src/dsp_tools/commands/project/create/project_create.py

```diff
@@ -230,13 +228,13 @@ def _get_group_iris_for_user(
                     f"exists on the DSP server, but no groups could be retrieved from the DSP server."
                 )
                 print(f"    WARNING: {err_msg}")
-                logger.warning(err_msg, exc_info=True)
+                logger.warning(err_msg)
                 success = False
                 continue
             existing_group = [g for g in remote_groups if g.project == current_project.iri and g.name == group_name]
             if not existing_group:
                 print(f"    WARNING: {inexisting_group_msg}")
-                logger.warning(inexisting_group_msg, exc_info=True)
+                logger.warning(inexisting_group_msg)
```

Comment:
the exc_info should not be necessary anymore
https://github.com/Delgan/loguru/issues/30

Reply:
That's not how I understand it. Rather, we should replace `logger.xyz(msg, exc_info=True)` with `logger.opt(exception=True).xyz(msg)`. See https://loguru.readthedocs.io/en/stable/resources/migration.html#replacing-exc-info-argument